tap_postgres/db.py

```diff
@@ -31,19 +33,24 @@ def calculate_destination_stream_name(stream, md_map):
     if include_schemas_in_destination_stream_name:
         return "{}_{}".format(md_map.get((), {}).get('schema-name'), stream['stream'])
 
-    return stream['stream']
+    return stream['tap_stream_id']
```

Comment:
do you need to fix the return above as well?

Reply:
`include_schemas_in_destination_stream_name` is hardcoded to false. Maybe we can remove it if we will never use it.

Reply:
Should I remove it right now?

Reply:
no that's fine